src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/AddPersonnelSideSheet/AddPersonnelFormDisciplinesDropDown.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { SearchableDropdown, TextInput } from '@equinor/fusion-components';
+import Personnel from '../../../../../../../models/Personnel';
+import { BasePosition } from '@equinor/fusion';
+
+export type PersonnelFormDisciplinesDropDown = {
+    selection: BasePosition[];
+    onChange: (changedPerson: Personnel) => void;
+    selectedField: string;
+    item: Personnel;
+    disabled: boolean;
+};
+
+const AddPersonnelFormDisciplinesDropDown: React.FC<PersonnelFormDisciplinesDropDown> = ({
+    selection,
+    onChange,
+    selectedField,
+    item,
+    disabled
+}) => {
+    const options = React.useMemo(() => {
+        if (!selection) return []
+
+        return selection.map(s => ({
+            title: s.name,
+            key: s.id,
+            isSelected: s.name === selectedField,
+        }));
+    }, [selection, selectedField]);
+
+    if (disabled)
+        return <TextInput
+            key={`disciplines${item.personnelId}`}
+            disabled={true}
+            placeholder={item.disciplines?.map(d => d.name).join('/') || ""}
+            onChange={() => { }}
+        />
+
+    return (
+        <SearchableDropdown
+            options={options}
+            onSelect={newValue => {
```

Comment:
The on select handler should be wrapped with useCallback (e.g. const onSelect = useCallback(..., [...]);) and remove console.log